ldes-server-port-ingest-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/rest/exception/MemberIdNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.ingest.rest.exception;
+
+import org.apache.jena.rdf.model.Model;
+
+public class MemberIdNotFoundException extends RuntimeException {
+
+	private final transient Model model;
+
+	public MemberIdNotFoundException(Model model) {
+		this.model = model;
+	}
+
+	@Override
+	public String getMessage() {
+		return "Member id could not be extracted from model: %n%n%s".formatted(model.listStatements().toList());
```

Comment:
Is this readable in the message? Wouldn't it be more readable if the model was parsed to turtle or n-quads?